README.md

```diff
@@ -1,20 +1,28 @@
 # <img src="https://i.imgur.com/nPCcxts.png" height=25> MCStatus
+
 [![discord chat](https://img.shields.io/discord/936788458939224094.svg?logo=Discord)](https://discord.gg/C2wX7zduxC)
 ![supported python versions](https://img.shields.io/pypi/pyversions/mcstatus.svg)
 [![current PyPI version](https://img.shields.io/pypi/v/mcstatus.svg)](https://pypi.org/project/mcstatus/)
+[![Docs](https://img.shields.io/readthedocs/mcstatus?label=Docs)](https://mcstatus.readthedocs.io/)
```

Comment:
`sphinx-docs` branch is still without a PR, so I would count it as draft. Should we push changes related to draft in master? Let's add this string in `sphinx-docs` branch instead.

Reply:
Follow this link and you will find that it works.

Reply:
Yes, but it's an empty page...
![image](https://user-images.githubusercontent.com/68118654/196970778-98a64624-1a55-42a6-93ab-cb8c0720a708.png)

Reply:
Yes, see pinned #136

Reply:
It is empty. There are no docs, furthermore it's confusing to see an empty page in "docs" link. I still think we should add link to docs, when we will add those docs.

Reply:
I still think that things which are draft, should be added in corresponding PR.

Reply:
This is the final version of what this link would look like, but I'll cherry-pick this into it's own PR